spec/rdkafka/producer_spec.rb

```diff
@@ -93,6 +114,36 @@ def call(report)
         expect(called_report.first.partition).to eq 1
         expect(called_report.first.offset).to be >= 0
       end
+
+      it "should provide handle" do
+        callback_handle = []
```

Comment:
Should be plural - `callback_handles`

Reply:
I've followed the pattern from the previous test: `called_report = []`, the array is used as a "reference wrapper" of sorts to capture the provided handle.

Reply:
Ah, I see. Maybe it is ok for consistency. In general it it would make sense to name it plural, because it can be mistaken for callback handle itself.

Reply:
Hi @gvisokinskas thanks for the PR! I agree with @laurynas, let's make it plural. We can change `called_report` to also use plural in another PR.